### ISSUE TITLE
Debug message on course overview block

### ISSUE THREAD

**Issue by abias** [academic-moodle-cooperation/moodle-mod_organizer#16]
Hi,

we are running the latest version of mod_organizer on Moodle 3.0.

On the MyMoodle page of our users, there is a custom course overview block which basically behaves like block_course_overview from Moodle core. This block especially fetches and displays course news from activities via the *_print_overview() functions, i.e. it calls organizer_print_overview().

Now we see that some users who have course news for an organizer activity are also getting a debug message in the course overview block saying:

```
Strict Standards: Only variables should be passed by reference in /var/www/html/moodle_ng/mod/organizer/lib.php on line 570
```

Are you able to track down the reason for this debug message and fix it?

Thanks,
Alex

**Comment by mkemmerling** [comment]
Hi Alex,

I forwarded the issue to the plugin developer.

Cheers,
Markus

**Comment by bluetom** [comment]
Hi Alex,

I was not able to reproduce the debug message with the moodle core course overview block. Anyway, in the meantime I have changed the code of lib.php at line 570 slightly.

Could you please try to reproduce the debug message with the latest organizer 3.0 release (which I think you have already installed)? Maybe the line number in the debug message has changed at least....that would give me a hint.

Thanks,
Thomas

**Comment by abias** [comment]
Hi Thomas,

thanks for your feedback.

I checked the debug message again on our staging system with the latest organizer release. The debug message is gone - but I can't tell for sure if this is caused by the organizer update or because I did anything else in our sandbox courses where the problem appeared.

Anyway, I think this issue can be closed and I will come back to you if the message re-appears.

Thanks for your effort.
Alex
